WordPress/twentytwentyfive
# Add screenshot

**Description**

Adds screenshot provided in #31.


**Screenshots**


![CleanShot 2024-08-19 at 17 49 23@2x](https://github.com/user-attachments/assets/d3d187ba-a9da-41eb-85c6-483f4c885ce9)


**Testing Instructions**

1. Check out this PR
2. Click on Appearance in backend
3. See screenshot being applied to this theme
